messages/src/message.rs

```diff
@@ -477,6 +491,23 @@ pub enum SpResponse {
         /// The modules on which fetching the LED state failed.
         failed_modules: ModuleId,
     },
+
+    /// The extended status of a set of transceiver modules.
```

Comment:
On the naming question, I think I prefer in order:

- Renaming `Status` to `HardwareStatus` or similar and making this `Status`.
- Keeping `Status` as is, and making this `ExtendedStatus`
- Anything else :)

Reply:
addressed in https://github.com/oxidecomputer/transceiver-control/pull/113#issuecomment-1610017071